src/VecSim/spaces/IP/IP_AVX512_FP32.cpp

```diff
@@ -48,7 +47,6 @@ float FP32_InnerProductSIMD4Ext_AVX512_impl(const void *pVect1v, const void *pVe
         __m512 v2 = _mm512_loadu_ps(pVect2);
```

Comment:
line 36 is redundant
and i think its a better practice to move line 31 to be above 64